component/searchBar.jsx

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+
+export function SearchBar({ onSubmit }) {
+  const [search, setSearch] = useState('');
+
+  return (
+    <StyledSearchbar>
+      <input
+        placeholder="Search Book"
+        type="text"
+        value={search.toUpperCase()}
+        onChange={event => {
+          setSearch(event.target.value);
+        }}
+      />
+      <button
+        type="submit"
+        onClick={() => {
+          onSubmit(search);
```

Comment:
At the moment there is no functionality here, I guess it is intended.

Reply:
I should have create another branch for the searchbar